### PR TITLE
Prepare class attributes and common methods

### PR DESCRIPTION
**Pull request by MichalSzewczyk2**
Added class attributes and common methods

### CODE REVIEW

File: src/visit/Visit.java [hunk 1]
```diff
@@ -1,4 +1,82 @@
...
+import java.util.Date;
+import java.util.Objects;
+
 public class Visit {
+
+    private LocalDate date;
+    private LocalTime time;
+    private int duration;
+    private Doctor doctor;
+    private Patient patient;
```
Comment (Goorsky1) [inline src/visit/Visit.java:17]: add `private int Rating`
accepts int values from 1 to 5 (less than1 -> 1, more than 5->5 , add this logic to setter)

Comment (Goorsky1) [inline src/visit/Visit.java:17]: add private Status stasus; (look it up on main branch)

File: src/users/Doctor.java [hunk 2]
```diff
@@ -1,4 +1,53 @@
...
-public class Doctor {
+import src.clinic.Clinic;
+
+import java.util.List;
+import java.util.Objects;
+
+public class Doctor extends User{
+
+
+    private List<Integer> expertiseId;
```
Comment (Goorsky1) [inline src/users/Doctor.java:11]: create `Expertise` class {
private int expertiseId;
private String expertiseName;
}

replace this List with `private HashSet<Expertise> expertises

File: src/users/Doctor.java [hunk 3]
```diff
@@ -1,4 +1,53 @@
...
+    private List<Integer> clinicId;
```
Comment (Goorsky1) [inline src/users/Doctor.java:12]: replace with ``HashSet<Clinic> clinics`

**Review by Wojoxdxd — APPROVED** [review]

**Review by Goorsky1 — APPROVED** [review]

**Pull request merged**